applications/admin/application/form.py

```diff
@@ -45,23 +78,7 @@ def __init__(self, *args, **kwargs) -> None:
 
     class Meta:
         model = Application
-        fields = [
-            "id",
-            "applicant_type",
-            "status",
-            "application_round",
-            "organisation",
-            "contact_person",
-            "user",
-            "billing_address",
-            "home_city",
-            "additional_information",
-            "working_memo",
-            "sent_date",
-            "cancelled_date",
-            "in_allocation_notification_sent_date",
-            "results_ready_notification_sent_date",
-        ]
+        fields = []  # Use fields from ModelAdmin
```

Comment:
Not a fan of this. Since the admin detail view is just for the model form, it should be the form that defines which fields are shown in the form (separation of concerns). Maybe this is not possible due to using fieldsets...

Reply:
Seems like this can't be resolved without introducing some replacement for ModelForm. We can live with this.